ted/src/main/java/frc/robot/Robot.java

```diff
@@ -16,6 +23,28 @@ public class Robot extends TimedRobot {
   @Override
   public void robotInit() {
     m_robotContainer = new RobotContainer();
+
```

Comment:
I think you put some code in Robot.java just to debug. But for the final check-in, there shouldn't be any changes in Robot.java Can you figure out if these changes are needed and if so move then where they belong (likely in the the LED subsystem)?